components/script/dom/webidls/Performance.webidl

```diff
@@ -49,3 +49,11 @@ partial interface Performance {
 partial interface Performance {
   PerformanceNavigationTiming timing();
 };
+// https://w3c.github.io/navigation-timing/#extensions-to-the-performance-interface
+[Exposed=Window]
```

Comment:
What if you move this Exposed annotation to the attribute instead, like:
```idl
partial interface Performance {
  [SameObject, Exposed=Window]
  readonly attribute PerformanceNavigation navigation;
}
```
If you run `./mach test-wpt tests/wpt/web-platform-tests/workers/worker-performance.worker.html` then there should not be any unexpected results.